content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -173,6 +173,38 @@ Two ways to avoid this:
 </LanguageSection>
 
 ### Dynamic Arrays
+<PySection>
+
+
+
+```py
+import sys
+
+n = int(input())
+a = [int(i) for i in input().strip().split(' ')]
```

Comment:
The `str.split()` method has a default argument of `' '` already, so you don't need it here.